styles/main.css

```diff
@@ -146,11 +146,33 @@ img.h1 {
     display: block;
 }
 
-img.hero {
-    width: 55%;
-    min-width: 288px;
-    display: block;
-    margin: 0 auto;
+/******************
+* Responsive Hero *
+******************/
+.hero {
+    padding-top: 0;
+    height: 176px;
+    background: url(../images/responsive-images/notebook-small.png) 50% /contain no-repeat;
+}
+
+@media all and (min-width: 530px) {
+    .hero {
```

Comment:
I looked at the smallest size the current notebook.png scales to when you go down to a screen width of 320px. I doubled that image size approximately, and created notebook-small.png to accomodate 2x pixel density screens. I picked that first break point (min-width: 530px) where notebook-small.png would no longer fit the needs of a 2x screen. From there I made two more breakpoints and images that I thought would be a good fit for different screen sizes. I didn't check to see if those breakpoints were used elsewhere.